### DISCUSSION TITLE
Condensed view of code-centric workflow

### DISCUSSION THREAD

**Discussion by xgdgsc** [fonsp/Pluto.jl#2357] [Pluto development]
Is it possible to provide something like the previous Juno extension that only shows a expandable result at line end to see more lines of code instead of results?  Currently there even isn' t a hide output option, only option to hide code. Usually when you first do explorational research, code is the index key, output is the value. You may be too lazy to write a title for every plot. Putting results above is already really bad https://github.com/fonsp/Pluto.jl/issues/205

**Comment by fonsp** [comment]
Hi @xgdgsc !

That's currently not possible. Though it is possible to hide code, by ending your cell's code with a `;` semicolon.
